server/src/main/java/org/elasticsearch/action/bulk/BulkFeatures.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.action.bulk;
+
+import org.elasticsearch.features.FeatureSpecification;
+import org.elasticsearch.features.NodeFeature;
+
+import java.util.Set;
+
+import static org.elasticsearch.action.bulk.TransportSimulateBulkAction.SIMULATE_MAPPING_VALIDATION;
+
+public class BulkFeatures implements FeatureSpecification {
```

Comment:
Should we call this `SimulateBulkFeatures` since it's not really for bulk itself?

Reply:
I am guessing that we'll have to add features for other bulk-api things at some point down the line, and there didn't seem to be any value in having more than one FeatureSpecification per package. This stuff is all new though so there's not really much precedent. What do you think?

Reply:
I mean, I don't feel super strongly about it, we can always change it later, so whichever you prefer